examples/snowpack/pay.js

```diff
@@ -0,0 +1,19 @@
+import * as Square from '@square/web-sdk';
+
+const applicationId = 'sandbox-sq0idb-lybe_WkfKNAbb3WklswmwA';
+const locationId = 'LKYXSPGPXK05M';
+
+async function start() {
+  const payments = await Square.payments(applicationId, locationId);
+  const card = await payments.card();
```

Comment:
Can we add some sort of error handling for the `await` statements. I think it is very important that we add it for the `tokenize` call.